api/v1alpha3/conversion.go

```diff
@@ -132,11 +153,15 @@ func (dst *MachineHealthCheck) ConvertFrom(srcRaw conversion.Hub) error {
 func (src *MachineHealthCheckList) ConvertTo(dstRaw conversion.Hub) error {
 	dst := dstRaw.(*v1alpha4.MachineHealthCheckList)
 
-	return  Convert_v1alpha3_MachineHealthCheckList_To_v1alpha4_MachineHealthCheckList(src, dst, nil)
+	return Convert_v1alpha3_MachineHealthCheckList_To_v1alpha4_MachineHealthCheckList(src, dst, nil)
 }
 
 func (dst *MachineHealthCheckList) ConvertFrom(srcRaw conversion.Hub) error {
 	src := srcRaw.(*v1alpha4.MachineHealthCheckList)
 
 	return Convert_v1alpha4_MachineHealthCheckList_To_v1alpha3_MachineHealthCheckList(src, dst, nil)
 }
+
+func Convert_v1alpha3_ClusterStatus_To_v1alpha4_ClusterStatus(in *ClusterStatus, out *v1alpha4.ClusterStatus, s apiconversion.Scope) error {
+	return autoConvert_v1alpha3_ClusterStatus_To_v1alpha4_ClusterStatus(in, out, s)
```

Comment:
Let me know if you'd prefer I do the ControlPlaneInitialized conversion in here. It's technically more accurate, but I wanted to use the `conditions.MarkTrue()` helper, and that requires a full root-level object (i.e. `Cluster`).